defrag/modules/db.py

```diff
@@ -0,0 +1,86 @@
+from os import environ as env
+from typing import Optional, Union
+from redis import Redis, BlockingConnectionPool
+from redis.client import Pipeline
+from pottery import RedisDict
+
+__MOD_NAME__ = "db"
```

Comment:
I do not want to have things like this as a module. I'd suggest creating a `modules/db` folder and placing it in there, or putting it in `helpers` or `utils`

Reply:
did that